src/test/kotlin/BuyerTest.kt

```diff
@@ -1,15 +1,40 @@
+import exception.ErrorCode
```

Comment:
테스트 클래스는 테스트 대상 클래스와 같은 패키지에 위치하면 어떨까요?

```suggestion
package model

import exception.ErrorCode
```